consensus/wal.go

```diff
@@ -17,6 +17,10 @@ import (
 	cmn "github.com/tendermint/tmlibs/common"
 )
 
+const (
+	maxMsgSizeBytes = 10024 // 10MB
```

Comment:
Is this correct estimation?

Reply:
shouldnt it be 10240 ? 

Also its based on the block part size. block parts are the largest messages we write, and currently default to 64kB, so this is more than enough!

Reply:
oh, forgot about this. yeah, will set to something lower (1MB) and add a comment.